hawc/apps/riskofbias/serializers.py

```diff
@@ -321,33 +321,15 @@ def validate(self, data):
 
     @transaction.atomic
     def create(self, validated_data):
-        rob = models.RiskOfBias.objects.create(
-            study=validated_data["study"],
-            final=validated_data["final"],
-            author=validated_data["author"],
-            active=validated_data["active"],
-        )
-
-        scores_to_create = validated_data["scores"]
-        for score_to_create in scores_to_create:
-            try:
-                score = models.RiskOfBiasScore.objects.create(
-                    riskofbias=rob,
-                    metric=score_to_create["metric"],
-                    is_default=score_to_create["is_default"],
-                    label=score_to_create["label"],
-                    score=score_to_create["score"],
-                    notes=score_to_create["notes"],
-                )
-            except ValidationError as err:
```

Comment:
Is it fine to not include this try/except block? ie will a Django ValidationError automatically be converted to the serializer version?

Reply:
I added it back just to be safe. We didn't have a test either so it's probably safer to keep it, though I don't think the `create` method runs the validators.